src/order/category/Frame.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.Lattice
+import order.hom.complete_lattice
+
+/-!
+# The category of frames
+
+This file defines `Frame`, the category of frames.
+
+## References
+
+* [nLab, *Frm*][http://nlab-pages.s3.us-east-2.amazonaws.com/nlab/show/Frm]
```

Comment:
Where are you getting these `amazonaws` urls from?

Reply:
That's how they show up on my side. Ask Bhavik, he's experiencing the same DNS weirdness as me.

Reply:
Something odd has been happening with nlab urls recently, but I think the more normal looking ones work, even if they sometimes redirect to the odd ones:
https://ncatlab.org/nlab/show/frame

Reply:
```suggestion
* [nLab, *Frm*][https://ncatlab.org/nlab/show/Frm]
```